src/dune/coq_lib_name.ml

```diff
@@ -1,29 +1,32 @@
 (* This file is licensed under The MIT License *)
 (* (c) MINES ParisTech 2018-2019               *)
 (* (c) INRIA 2020                              *)
-(*     Written by: Emilio Jesús Gallego Arias  *)
+(* Written by: Emilio Jesús Gallego Arias *)
 
 open! Stdune
+
 (* Coq Directoy Path *)
 type t = string list
 
 let to_string x = String.concat ~sep:"." x
+
 let wrapper x = to_string x
 
 let decode : (Loc.t * t) Dune_lang.Decoder.t =
-  Dune_lang.Decoder.plain_string (fun ~loc s ->
-    (loc, String.split ~on:'.' s))
+  Dune_lang.Decoder.plain_string (fun ~loc s -> (loc, String.split ~on:'.' s))
 
 let encode : t Dune_lang.Encoder.t =
-  fun lib -> Dune_lang.Encoder.string (to_string lib)
+ fun lib -> Dune_lang.Encoder.string (to_string lib)
```

Comment:
Is this intended?

Reply:
This is. This would format the same with 0.12 or 0.13.